src/main/java/edu/harvard/iq/dataverse/settings/FeatureFlags.java

```diff
@@ -85,6 +85,19 @@ public enum FeatureFlags {
      */
     API_BEARER_AUTH_USE_SHIB_USER_ON_ID_MATCH("api-bearer-auth-use-shib-user-on-id-match"),
 
+    /**
+     * Allows the use of an OAuth user account (GitHub, Google, or ORCID) when an identity match is found during API bearer authentication.
+     * This feature enables automatic association of an incoming IdP identity with an existing OAuth user account,
+     * bypassing the need for additional user registration steps.
+     *
+     * <p>The value of this feature flag is only considered when the feature flag
+     * {@link #API_BEARER_AUTH} is enabled.</p>
+     *
+     * @apiNote Raise flag by setting "dataverse.feature.api-bearer-auth-use-oauth-user-on-id-match"
+     * @since Dataverse @TODO:
```

Comment:
```suggestion
     * @since Dataverse 6.8:
```

Seems weird to have a colon instead of a period but whatever, it's consistent with the other ones! 😄